Mergin/utils.py

```diff
@@ -408,13 +408,12 @@ def get_new_qgis_project_filepath(project_name=None):
     return None
 
 
-def unsaved_project_check(force_saving=False):
+def unsaved_project_check():
     """
     Check if current QGIS project has some unsaved changes.
     Let the user decide if the changes are to be saved before continuing.
-    :return: True if previous method should continue, False otherwise.
-    If 'force_saving' flag is set to True then changes have to be saved before continuing.
-    :type: boolean
+    :return: True if previous method should continue, False otherwise and replay button (or None if no changes detected)
```

Comment:
Anothe way would be to return some enum value for each state: has unsaved changes, no unsaved changes, has changes but ignore.